tests/test_fourier.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+from iospi import fourier
+
+
+# right way to use as global in testing environment?
```

Comment:
Yes I think this is fine for now, but maybe remove this comment?

Reply:
ok